### DISCUSSION TITLE
Using newer versions of Electron

### DISCUSSION THREAD

**Discussion by teslor** [Deluze/electron-vue-template#61] [Q&A] [answered]
Does it support Electron v28-30 and (if yes) is it safe to use? Electron v27 (current dependency) is a bit outdated.

**Comment by Deluze** [comment, marked as answer]
As this template is very bare-bones, it's usually very safe to upgrade to a new major version. At least, we haven't had any breaking changes coming in with the last major releases.

Feel free to update Electron in your own project. I agree that the Electron version in this template is a bit outdated. An update to the latest version would be nice :-)
